pkg/messaging/grpc_proxy.go

```diff
@@ -112,7 +112,7 @@ func (p *proxy) intercept(ctx context.Context, fullName string) (context.Context
 	if isLocal {
 		// proxy locally to the app
 		if p.acl != nil {
-			ok, authError := acl.ApplyAccessControlPolicies(ctx, fullName, common.HTTPExtension_NONE, config.GRPCProtocol, p.acl) //nolint:nosnakecase
+			ok, authError := acl.ApplyAccessControlPolicies(ctx, fullName, common.HTTPExtension_NONE, false, p.acl) //nolint:nosnakecase
```

Comment:
Always false?

Reply:
The argument has changed. Before it was the protocol name, either "http" or "grpc", and we were passing "grpc". Now the argument is "isHTTP" so in this case it's always false.